CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Next release
 
+- dev docs: add documentation to the rocksdb configuration
```

Comment:
Not related to this PR, but you know something is not quite right when you have to update the changelog for developer comments!

Reply:
that's right!
ive said it offline plenty of times: if the changelog is a copy of the git log, then the changelog has absolutely no purpose

the changelog should be user-facing and non-verbose with migration notes and plenty of details for all the new features. It's probably best written by looking at the git log just before a release

The [bevy changelog](https://bevyengine.org/news/bevy-0-14/) is a very good example of a very well made changelog - imo, it's actually the best example ive ever seen of a changelog, it's always a delightful read and probably gargantuan task to write it.

we don't do releases that often yet though, and we are not the same kind of software as bevy, we don't expect the same kind of audience - i would be in favor of just removing that changelog file and the github action for now, and probably do a feature-by-feature user facing changelog for real releases.

Reply:
I 100% second this!